v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/modbus_evse_client.py

```diff
@@ -1001,17 +1087,12 @@ async def __cancel_polling(self, reason: str = ""):
             reason (str, optional): For debugging only
         """
         if not self._am_i_active:
-            self.__log(
-                "__cancel_polling called while _am_i_active == False. Not blocking."
-            )
+            self.__log("called while _am_i_active == False. Not blocking.")
 
-        self.__log(f"__cancel_polling, reason: {reason}")
-        if self.hass.timer_running(self.poll_timer_handle):
-            await self.hass.cancel_timer(self.poll_timer_handle, True)
-            # To be really sure...
-            self.poll_timer_handle = None
-        else:
-            self.__log("__cancel_polling: No timer to cancel")
+        self.__log(f"reason: {reason}")
+        self.__cancel_timer(self.poll_timer_handle)
+        # To be really sure...
```

Comment:
Setting a timer to None after canceling it seems like common sense. So, I think adding this comment is very redundant.